sources.py

```diff
@@ -434,6 +437,11 @@ def _get_data(self):
         else:
             logger.debug(f"{self.IDENTIFIER}: cache miss for {req.url}")
 
+        if res.from_cache:
+            print(f"Cache hit for {req.url}")
+        else:
+            print(f"Cache miss for {req.url}")
```

Comment:
We either want to remove these or move the to `logger.debug`. I'm not sure if I had removed this earlier, could check `main`.